src/components/see-more-titles/see-more-titles.module.scss

```diff
@@ -0,0 +1,12 @@
+.arrow {
+  position: fixed;
+  top: 14%;
+  width: 20.3%;
+  margin-right: 5%;
+  cursor: pointer;
+
+  svg {
+    width: auto;
+    height: auto;
+  }
```

Comment:
I think this is very custom where you use it, the component is simple without icons. Maybe you can define this in the component that calls SeeMoreTitles

Reply:
Done